<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperationInterface</class>
 <widget class="QWidget" name="OperationInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OperationInterface</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>工号：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="UserEdit">
      <property name="text">
       <string>10030534</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>密码：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="PasswordEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Qwer.12345</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="LoginPushButton">
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SavePasswordPushButton">
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="3" rowspan="9">
    <widget class="Line" name="line_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>项目路径:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLineEdit" name="MobanlujinEdit"/>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>项目名称:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QComboBox" name="PathcomboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="7" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="7">
      <widget class="QLabel" name="label_jiancexiangmu">
       <property name="text">
        <string>请先输入mms子单号</string>
       </property>
      </widget>
     </item>
     <item row="6" column="6">
      <widget class="QLabel" name="label_47">
       <property name="text">
        <string>模穴号:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_30">
       <property name="text">
        <string>项目名称：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLabel" name="label_shenqingdanhao">
       <property name="text">
        <string>请先输入mms子单号</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_36">
       <property name="text">
        <string>检测要求：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4" colspan="2">
      <widget class="QLabel" name="label_jitai">
       <property name="text">
        <string>请先输入mms子单号</string>
       </property>
      </widget>
     </item>
     <item row="6" column="7">
      <widget class="QLabel" name="label_muxuehao">
       <property name="text">
        <string>请先输入mms子单号</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QLabel" name="label_29">
       <property name="text">
        <string>申请时间:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>送检单位:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>申请单号：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4" colspan="2">
      <widget class="QLabel" name="label_yangpingshuliang">
       <property name="text">
        <string>请先输入mms子单号</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="NumberEdit">
       <property name="text">
        <string>mmsdCY12410231B2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="label_32">
       <property name="text">
        <string>样品数量：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="GetInfoButton">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
     <item row="4" column="7">
      <widget class="QLabel" name="label_shoujianshijian">
       <property name="text">
        <string>请先输入mms子单号</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLabel" name="label_38">
       <property name="text">
        <string>是否破坏：</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QLabel" name="label_44">
       <property name="text">
        <string>夹具编号：</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QLabel" name="label_gongxu">
       <property name="text">
        <string>请先输入mms子单号</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QLabel" name="label_jianyanleibie">
       <property name="text">
        <string>请先输入mms子单号</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4" colspan="2">
      <widget class="QLabel" name="lable_shenqingrenyuan">
       <property name="text">
        <string>请先输入mms子单号</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>申请人员：</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_beizhu">
       <property name="text">
        <string>备注：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QLabel" name="label_shenqingshijian">
       <property name="text">
        <string>请先输入mms子单号</string>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QLabel" name="label_songjiandanwei">
       <property name="text">
        <string>请先输入mms子单号</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="8">
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="4" colspan="2">
      <widget class="QLabel" name="label_shifoupohuai">
       <property name="text">
        <string>请先输入mms子单号</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLabel" name="label_jianceyaoqiu">
       <property name="text">
        <string>请先输入mms子单号</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_48">
       <property name="text">
        <string>工序：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLabel" name="label_xiangmmumingchen">
       <property name="text">
        <string>请先输入mms子单号</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_42">
       <property name="text">
        <string>检验类别：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>机台：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="6">
      <widget class="QLabel" name="label_41">
       <property name="text">
        <string>检测项目:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="4" colspan="2">
      <widget class="QLabel" name="label_jiajubianhao">
       <property name="text">
        <string>请先输入mms子单号</string>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QLabel" name="label_35">
       <property name="text">
        <string>收件时间:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>mm子单号：</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>材料名称：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QComboBox" name="TypecomboBox">
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>报告选择:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QComboBox" name="FilecomboBox">
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>开始时间</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QDateTimeEdit" name="dateTimeEditStart">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="currentSectionIndex">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>结束时间</string>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QDateTimeEdit" name="dateTimeEditEnd">
     <property name="displayFormat">
      <string>yyyy-M-d H:mm</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>测量人员</string>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="7" column="4">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>审核人员</string>
     </property>
    </widget>
   </item>
   <item row="7" column="5">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="8" column="6">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="4">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>iP：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="IpEdit">
         <property name="text">
          <string>192.168.31.145</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>端口：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="PortEdit">
         <property name="text">
          <string>8889</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="connectButton">
         <property name="text">
          <string>建立链接</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="GetDataButton">
         <property name="text">
          <string>获取数据</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="11" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>采集方式：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="caijiTypecomboBox">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>数据对接</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>文件感知</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>设备品牌：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sehbeicomboBox">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>兆丰1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>兆丰2</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="11" column="2" rowspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QPushButton" name="ShowDataButton">
       <property name="text">
        <string>信息显示</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="ComCheckButton">
       <property name="text">
        <string>完成检测</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="WriteFilepushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>上传数据</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="ClearDataButton">
       <property name="text">
        <string>清空数据</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="12" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>自动感知文件路径：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
    </layout>
   </item>
   <item row="13" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>文件保存路径：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
    </layout>
   </item>
   <item row="14" column="0" colspan="7">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="15" column="0" colspan="7">
    <widget class="QTextEdit" name="showdataEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
